_docs/concepts/network-and-auth/auth.md

```diff
@@ -0,0 +1,150 @@
+type: markdown
+---
+
+## Overview
+
+Istio Auth's aim is to enhance the security of microservices and their communication without requiring service code changes. It is responsible for:
+
+
+
+*   Providing each service with a strong identity that represents its role to enable interoperability across clusters and clouds
+
+*   Securing both service to service communication and end-user to service communication
+
+*   Providing a key management system to automate key/cert generation, distribution, rotation, and revocation
+
+In future versions it will also provide:
+
+*   Fine-grained authorization and auditing to control and monitor who accesses your services, apis, or resources
+
+*   Powerful authorization mechanisms: [ABAC](https://en.wikipedia.org/wiki/Attribute-Based_Access_Control), [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control), Authorization hooks
+
+## Architecture
+
+The figure below shows the Istio Auth architecture, which includes three important components: identity, key management, and communication security. It describes how Istio Auth is used to secure the service-to-service communication between service A, running as the service account "foo", and service B, running as the service account "bar".
+
+<img style="display:block;margin:auto" src="./img/auth/auth.svg" alt="Istio Auth Architecture." />
+
+## Components
+
+### Identity
+
+Istio Auth uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) to identify who runs the service because of the following reasons:
+
+
+
+*   A service account is **the identity (or role) a workload runs as**, which represents that workload's privileges. For the system requiring strong security, the amount of privilege for a workload should not be identified by a random string (i.e., service name, label, etc), or by the binary that is deployed.
+
+    *   For example, let's say we have a workload pulling data from a multi-tenant database. If Alice ran this workload, she will be able to pull a different set of data than if Bob ran this workload.
+
+*   Service accounts enable powerful security policies by offering the flexibility to identify a machine, a user, a workload, or a group of workloads (different workloads can run as the same service account).
+
+*   The service account a workload runs as won't change during the lifetime of the workload.
+
+*   Service account uniqueness can be ensured with domain name constraint
+
+### Communication Security
+
+Service-to-service communication is tunneled through the client side [Envoy](https://lyft.github.io/envoy/) and the server side Envoy. The end-to-end communication is secured by:
+
+
+
+*   Local TCP connections between the service and Envoy
+
+*   Mutual TLS connections between proxies
+
+*   Secure Naming: during the handshake process, the client side Envoy checks that the service account provided by the server side certificate is allowed to run the target service
```

Comment:
Is 'target service' correct? I'm guessing the same checks apply for ingress traffic too, which would make the other side the 'source service' instead.